lib/mutant/mutator/node/generic.rb

```diff
@@ -37,17 +37,18 @@ class Generic < self
         ]
 
         unsupported_regexp_nodes = AST::Types::REGEXP.to_a - %i[
-          regexp_root_expression
+          regexp_alternation_meta
           regexp_bol_anchor
-          regexp_word_type
-          regexp_nonword_type
           regexp_digit_type
+          regexp_eol_anchor
```

Comment:
This one was added, but I decided to sort the rest of the list because I was not sure where to insert it.